testsuite/tests/kuadrant/authorino/wristband/conftest.py

```diff
@@ -0,0 +1,110 @@
+"""Conftest for Edge Authentication tests"""
+import pytest
+
+from testsuite.openshift.objects.auth_config import AuthConfig
+from testsuite.certificates import CertInfo
+from testsuite.objects import Authorization
+from testsuite.utils import cert_builder
+
+
```

Comment:
There is a missing `run_on_kuadrant` fixture (should return False), as you need custom envoy